AElf.Kernel/Account.cs

```diff
@@ -2,20 +2,18 @@
 
 namespace AElf.Kernel
 {
-    /// <summary>
-    /// Account.
-    /// </summary>
     public class Account : IAccount
     {
-        private Address m_address;
+        public int Amount { get; set; }
 
-        Account(Address address) {
-            this.m_address = address;
+        public IHash<IAccount> GetAddress()
+        {
+            return new Hash<IAccount>(this.GetSHA256Hash());
```

Comment:
Maybe we need to just call this.GetHash();

Reply:
100%